<?xml version='1.0'?>
<launch>

	<!-- Load the cube -->
	<param name='cube_description' command="$(find xacro)/xacro --inorder '$(find dobot_description)/urdf/cube_1.urdf'" />
	<param name='use_gui' value='true' />

	<!-- Spawn the cube in Gazebo -->
	<node name='spawn_model' pkg='gazebo_ros' type='spawn_model' respawn='false' output='screen' 
		 args='-urdf -model cube_1 -param cube_description' />

</launch>